scripts/scripts_object_storage/openstack_swift/local-setup-spark.sh

```diff
@@ -0,0 +1,10 @@
+ #!/usr/bin/env bash
+# Copyright 2016 Yahoo Inc.
+# Licensed under the terms of the Apache 2.0 license.
+# Please see LICENSE file in the project root for terms.
+#
+# This script install Spark locally
+
+wget http://archive.apache.org/dist/spark/spark-2.0.1/spark-2.0.1-bin-hadoop2.7.tgz
```

Comment:
Yes, exactly. I spent a few minutes trying to figure out which pom.xml file to target and where to put those in.

About profiles, to be frank, I am not well versed with that. Could you please elaborate ? I'm still in the learning phase.

Thanks,
Arun